src/clj/puppetlabs/services/ca/certificate_authority_disabled_service.clj

```diff
@@ -21,6 +21,6 @@
     [this localcacrl]
     (log/info "CA disabled; ignoring retrieval of CA CRL"))
 
-  (get-oid-mappings
+  (get-auth-handler
     [this]
-    {}))
+    (fn [request] (wrap-with-authorization-check request {:oid-map {}}))))
```

Comment:
Oh, I think I understand.  In that case, I think perhaps this line should change to simply return `wrap-with-authorization-check`.